Shift3/boilerplate-client-react
# feat(Testing-Library): Add ESLint plugins

## Changes
Added the recommended plugins for React Testing Library:
- ESLint-plugin-testing-library ('https://github.com/testing-library/eslint-plugin-testing-library')
- ESLint-plugin-jestdom ('https://github.com/testing-library/eslint-plugin-jest-dom')

## Purpose
To avoid possible mistakes when using the React Testing Library, these plugins are recommended. Please review this article from the creator of React Testing Library, for context ('https://kentcdodds.com/blog/common-mistakes-with-react-testing-library')

## Approach
[`Yarn add  --dev eslint-plugin-testing-library eslint-plugin-jest-dom`]

## Testing
1. Pull in the changes to your local copy of this branch and run [`yarn start`]
2. Make sure the project runs with no errors.

### Closes #102
